demo/admin/src/common/MasterMenu.tsx

```diff
@@ -1,17 +1,19 @@
 import { Menu, MenuCollapsibleItem, MenuContext, MenuItemRouterLink, useWindowSize } from "@comet/admin";
-import { Assets, Dashboard, Data, PageTree, Snips, Wrench } from "@comet/admin-icons";
+import { getMenuFromRouteMenu } from "@comet/cms-admin";
 import * as React from "react";
-import { useIntl } from "react-intl";
 import { useRouteMatch } from "react-router";
 
+import { routeMenu } from "./routeMenu";
+
 const permanentMenuMinWidth = 1024;
 
 const MasterMenu: React.FC = () => {
     const { open, toggleOpen } = React.useContext(MenuContext);
     const windowSize = useWindowSize();
-    const intl = useIntl();
     const match = useRouteMatch();
 
+    const menu = getMenuFromRouteMenu(routeMenu);
```

Comment:
This variable doesn't change depending on any React state, so you could define outside of the React component.

Reply:
it will once user permissions will get applied here :)

Reply:
You're right, haven't thought about that.